cs-rin-ru-enhanced-mod.user.js

```diff
@@ -48,6 +49,8 @@ const AJAX_LOADER = `
 
 const FORUM_NAME = 'CS.RIN.RU - Steam Underground Community';
 
+const navBarSize = 1.0 //In em
```

Comment:
If it should be in em then maybe we should just define it like that?
const navBarSize = "1em";

Reply:
Yeah, true. I go change that